tests/manage/test_ocs_324.py

```diff
@@ -92,11 +94,24 @@ def test_pvc_delete_create_same_name(self, test_fixture):
         global PVC_OBJ
 
         PVC_OBJ = helpers.create_pvc(sc_name=self.sc_obj.name)
-        logger.info(f"Deleting PersistentVolumeClaim with name {PVC_OBJ.name}")
+        pv_obj = ocp.OCP(
+            kind=constants.PV, namespace=config.ENV_DATA['cluster_namespace']
+        )
+        backed_pv = PVC_OBJ.get().get('spec').get('volumeName')
```

Comment:
backed_pv = PVC_OBJ.backed_pv

Reply:
Resolving based on the above conversation